src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -71,6 +71,8 @@ namespace Microsoft.Terminal.Settings.Model
         String EvaluatedStartingDirectory { get; };
 
         FontConfig FontInfo { get; };
+        INHERITABLE_PROFILE_SETTING(String, CellSizeX);
+        INHERITABLE_PROFILE_SETTING(String, CellSizeY);

```

Comment:
Yeah I'm absolutely open to such a discussion. The reason I didn't put it there is because I felt like cell sizes are unrelated to the font choice. For instance when you say that your cells are exactly 15px by 30px, then that doesn't involve the font at all, right?
But on the other hand it's of course traditionally associated with fonts... `cellSize.height` is basically `line-height` and `cellSize.width` is basically `letter-spacing` in CSS.

How do others feel about this?

Reply:
Correct me if I'm wrong, but I think the current implementation is like this:

|  | Current Impl | Proposed |
|--|--|--|
| `cellSize.height` and `cellSize.width` both defined in Profile.Defaults, not Profile | Profile inherits both | Profile inherits both |
| `cellSize.height` defined in Profile.Defaults, `cellSize.width` defined in Profile | Both defined in Profile | `cellSize.width` applied, `cellSize.height` NOT inherited |

Ideally, `cellSize` would be a child of `font` so that we would get this behavior for free, basically.
_However_, if you (plural you, so anybody that reads this really) don't agree with this being a part of the font object, at the very least I think we should apply the "all-or-nothing" heuristic here.

This is still the main thing holding me back from approving, so I'll throw the Needs-Discussion tag on this to make sure you get unblocked :)